tensorflow/contrib/gan/python/train_test.py

```diff
@@ -394,6 +395,24 @@ def test_grad_penalty_acgan(self):
   def test_grad_penalty_callable_acgan(self):
     self._test_grad_penalty_helper(create_callable_acgan_model)
 
+  def test_grad_penalty_one_sided_gan(self):
```

Comment:
I strongly applaud your desire to write good tests, but `_test_grad_penalty_helper` only checks that the result is the right kind of object. We don't check a multiple options on other losses, so I don't see these tests as necessary. On the other hand, test-bloat is something to consider. @lgeiger do you have strong feelings on whether to keep or remove these tests? @rmlarsen @sguada do you have thoughts?

Reply:
I don't have strong feelings on this. I agree these tests mostly check that the arguments are correctly passed on (I don't know if pylint does this already). Should we keep one of them to prevent trivial regressions?

Reply:
I wold never argue in favor of removing tests :-)

Reply:
Ack. Let's keep them then!